apps/admin_buildpack_lifecycle.go

```diff
@@ -271,7 +271,7 @@ exit 1
 
 		Expect(push).To(Exit(0))
 		appOutput := cf.Cf("app", appName).Wait()
-		Expect(appOutput).To(Say("buildpacks?:.*\\n.+\\n.+\\n.*Simple"))
+		Expect(appOutput).To(Say("buildpacks?:.*Simple"))
```

Comment:
Don't we need `(?s)` to make `.` match `\n`?

Reply:
True, that would have been a better approach. But since the change in output from cf7 matches the new regexp, it should be ok.